halotools/mock_observables/pair_counters/double_tree.py

```diff
@@ -440,76 +443,76 @@ def set_tree2_cell_sizes(self,
         y2mult = min(max_cells_per_dimension, y2mult)
         z2mult = min(max_cells_per_dimension, z2mult)
 
-        modified_x2_cellsize = self.tree1.xcell_size/x2mult
-        modified_y2_cellsize = self.tree1.ycell_size/y2mult
-        modified_z2_cellsize = self.tree1.zcell_size/z2mult
+        modified_x2_cellsize = self.tree1.xcell_size / x2mult
```

Comment:
@aphearin  - you mean this one not being int division? If yes then this was intentional as I though this was a bug in the original code (based to the docstring these cellsizes should be floats, and indeed similar parameters in other functions were with float division). So I'm sorry if I got it wrong.